packages/core/src/mpi/match-patients.ts

```diff
@@ -175,3 +178,115 @@ function isSameIdentifierById(a?: PersonalIdentifier, b?: PersonalIdentifier): b
     (a.type === "driversLicense" && b.type === "driversLicense" ? a.state === b.state : true)
   );
 }
+
+export function epicMatchingAlgorithm(
```

Comment:
would be good to add a link to epic's matching algorithm for reference

Reply:
no such link exists. Its a random pdf. will add it to google drive and link that